src/util/db/sqlstringformatter.cpp

```diff
@@ -14,7 +14,7 @@ QString SqlStringFormatter::format(
         return value; // unformatted
     }
 #if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
-    QSqlField stringField(QString(), QMetaType(QMetaType::QString));
+    QSqlField stringField{QString(), QMetaType(QMetaType::QString)};
```

Comment:
@Swiftb0y These issues could also be prevented by consistently using `auto` for direct initialization as I proposed in recent discussions and chats:

```
auto stringField = QSqlField(QString(), QMetaType(QMetaType::QString));
```

(just a remark, nothing to do here)

Reply:
Not sure really. Initialization is a mess in C++: https://www.youtube.com/watch?v=7DTlWPgX6zs

Reply:
Using curly braces or parentheses for initialization is a different topic ;)